src/main/webapp/resources/js/apis/analysis/analysis.js

```diff
@@ -149,6 +149,77 @@ export async function getSistrResults(submissionId) {
   }
 }
```

Comment:
Looking at this file, can you rename the URLs at the top, maybe at `ANALYSIS_URL` and `ANALYSES_URL` just so it is clear what they are for.

Reply:
Updated in d57854982494fb9733f7243e6d74f315e1ee9297